web-console/src/components/rule-editor/rule-editor.spec.tsx

```diff
@@ -36,4 +36,64 @@ describe('rule editor', () => {
     const { container } = render(ruleEditor);
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('matches snapshot with non existing tier in rule', () => {
+    const ruleEditor = (
+      <RuleEditor
+        rule={{
+          type: 'loadByInterval',
+          period: '2010-01-01/2015-01-01',
+          tieredReplicants: { nonexist: 2 },
+        }}
+        tiers={['test1', 'test2', 'test3']}
+        onChange={() => {}}
+        onDelete={() => {}}
+        moveUp={null}
+        moveDown={null}
+      />
+    );
+    const { container } = render(ruleEditor);
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it('matches snapshot with existing tier in rule', () => {
+    const ruleEditor = (
+      <RuleEditor
+        rule={{
+          type: 'loadByInterval',
+          period: '2010-01-01/2015-01-01',
+          tieredReplicants: { test1: 2 },
+        }}
+        tiers={['test1', 'test2', 'test3']}
+        onChange={() => {}}
+        onDelete={() => {}}
+        moveUp={null}
+        moveDown={null}
+      />
+    );
+    const { container } = render(ruleEditor);
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+    it('matches snapshot with existing tier and non existing tier in rule', () => {
+      const ruleEditor = (
+        <RuleEditor
+          rule={{
+            type: 'loadByInterval',
+            period: '2010-01-01/2015-01-01',
+            tieredReplicants: {
+              test1: 2,
+              nonexist: 1,
+            },
+          }}
+          tiers={['test1', 'test2', 'test3']}
+          onChange={() => {}}
+          onDelete={() => {}}
+          moveUp={null}
+          moveDown={null}
+        />
+      );
+      const { container } = render(ruleEditor);
+      expect(container.firstChild).toMatchSnapshot();
+    });
```

Comment:
CI is flagging the extra indentation for this test
https://travis-ci.org/github/apache/druid/jobs/689087087#L411

Reply:
Done